letsencrypt-apache/letsencrypt_apache/tls_sni_01.py

```diff
@@ -1,4 +1,5 @@
-"""ApacheDVSNI"""
+"""A TLS-SNI-01 authenticator for Apache"""
```

Comment:
It doesn't? Huh! I thought it did, but it looks like I missed a few methods. I'll fix it.